testData/solutions/win32.json

```diff
@@ -12,13 +12,49 @@
             {
                 "type": "gpii.settingsHandlers.INISettingsHandler",
                 "options": {
-                    "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\15.0\\Settings\\VoiceProfiles.ini"
+                    "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\16.0\\Settings\\enu\\DEFAULT.JCF",
```

Comment:
On my German PC with a German version of JAWS 14, the file `DEFAULT.JCF` is located at `"${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\14.0\\Settings\\deu\\DEFAULT.JCF"`. The significant part is the `deu` (instead of `enu`).